apps/web/modules/settings/teams/[id]/event-types-view.tsx

```diff
@@ -18,7 +18,9 @@ export const CreateTeamEventType = () => {
   const teamId = searchParams?.get("id") ? Number(searchParams.get("id")) : -1;
 
   const onSuccessMutation = () => {
-    router.push(`/settings/teams/${teamId}/profile`);
+    const redirectUrl = localStorage.getItem("onBoardingRedirect");
+    localStorage.removeItem("onBoardingRedirect");
+    redirectUrl ? router.push(redirectUrl) : router.push(`/settings/teams/${teamId}/profile`);
```

Comment:
Adding handler for users who choose "Team" account type and then get routed to `settings/teams/new` to create a team